models/multilabel_model.py

```diff
@@ -0,0 +1,438 @@
+          else:
+              negative_filter = None
+          # Only use examples that have a label in label filter array
+
+          positive_example_select_vector = get_select_vector(df, positive_filter)
+          positive_examples_df = df.loc[positive_example_select_vector]
+
+          # This ensures there no overlap between positive and negative examples
+          negative_example_select_vector = ~positive_example_select_vector
+          if negative_filter is not None:
+            # Exclude even further examples that don't fall into the negative label filter
+            negative_example_select_vector &= get_select_vector(df, negative_filter)
+          negative_examples_df = df.loc[negative_example_select_vector]
+
+          # No longer need df after this point
+          #del df
+
+          train_positive_examples_df, test_positive_examples_df = \
+                    split_and_subsample_dataframe(dataframe=positive_examples_df,
+                                                  validation_split=validation_split,
+                                                  subsample=input_file_dict["subsample"])
+
+          del positive_examples_df
+
+          train_negative_examples_df, test_negative_examples_df = \
+                    split_and_subsample_dataframe(dataframe=negative_examples_df,
+                                                  validation_split=validation_split,
+                                                  subsample=input_file_dict["subsample"])
+
+          del negative_examples_df
+
+          # append to overall list of examples
+
+          list_of_train_dataframes += [train_positive_examples_df, train_negative_examples_df]
+
+
+
+          list_of_test_dataframes += [test_positive_examples_df, test_negative_examples_df]
+          k+=1
+
+    train_df = pd.concat(list_of_train_dataframes, ignore_index=True)
+    final_dfs_train.append(train_df)
+    test_df = pd.concat(list_of_test_dataframes, ignore_index=True)
+    final_dfs_test.append(test_df)
+
+    DF_TRAIN = pd.concat(final_dfs_train, ignore_index=True)
+    DF_TEST = pd.concat(final_dfs_test, ignore_index=True)
+    print("Import done.")
+
+    return DF_TRAIN, DF_TEST
+
+
+DF_TRAIN, DF_TEST = import_dataframes(dataframe_file_list=config["train"]["inputDataFrames"],
+                              positive_label_filter_arr=pos_sounds,
+                              negative_label_filter_arr=neg_sounds,
+                              validation_split=config["train"]["validationSplit"])
+
+
+
+
+LABELS_BINARIZED_TRAIN = pd.DataFrame()
+LABELS_BINARIZED_TEST = pd.DataFrame()
+for key in pos_sounds.keys():
+    FULL_NAME = key
+    POSITIVE_LABELS = pos_sounds[key]
+    LABELS_BINARIZED_TRAIN[FULL_NAME] = 1.0 * get_select_vector(DF_TRAIN, POSITIVE_LABELS)
+
+
+    LABELS_BINARIZED_TEST[FULL_NAME] = 1.0 * get_select_vector(DF_TEST, POSITIVE_LABELS)
+
+
+
+print("TR: ",LABELS_BINARIZED_TRAIN.columns)
+print("TS: ",LABELS_BINARIZED_TEST.columns)
```

Comment:
Please add the following code:
```suggestion
print("TS: ",LABELS_BINARIZED_TEST.columns)

TOTAL_TRAIN_EXAMPLES_BY_CLASS = LABELS_BINARIZED_TRAIN.sum(axis=0)
TOTAL_TEST_EXAMPLES_BY_CLASS = LABELS_BINARIZED_TEST.sum(axis=0)
TOTAL_TRAIN_TEST_EXAMPLES_BY_CLASS = TOTAL_TRAIN_EXAMPLES_BY_CLASS + TOTAL_TEST_EXAMPLES_BY_CLASS
TOTAL_TRAIN_TEST_EXAMPLES = LABELS_BINARIZED_TRAIN.shape[0] + LABELS_BINARIZED_TEST.shape[0]

print("TRAIN NUMBER EXAMPLES (BY CLASS):")
print(TOTAL_TRAIN_EXAMPLES_BY_CLASS)
print("TEST NUMBER EXAMPLES (BY CLASS):")
print(TOTAL_TRAIN_TEST_EXAMPLES_BY_CLASS)
print("TOTAL NUMBER EXAMPLES (BY CLASS):")
print(TOTAL_TRAIN_TEST_EXAMPLES_BY_CLASS)
print("PERCENT EXAMPLES (BY CLASS):")
print(TOTAL_TRAIN_TEST_EXAMPLES_BY_CLASS / TOTAL_TRAIN_TEST_EXAMPLES)
```